### ISSUE TITLE
Create a list of blog favorites

### ISSUE THREAD

**Issue by moxiegirl** [docker/swarm#1572]
See https://github.com/docker/swarm/issues/1023 for one entry. Find others.

**Comment by abronan** [comment]
That's cool :+1: 

I'll try to gather useful resources too and make a list! I have some already.

**Comment by moxiegirl** [comment]
Great!

**Comment by amitshukla** [comment]
@moxiegirl should this be assigned to @rolfedh ?

**Comment by moxiegirl** [comment]
@amitshukla Not now. Let's look at it for 1.20.

**Comment by amitshukla** [comment]
@moxiegirl any update on this?

**Comment by moxiegirl** [comment]
@amitshukla @abronan was going to gather a list.

**Comment by abronan** [comment]
I have a list of blog posts that can be really helpful for users. But I was thinking, maybe we should do it in another repository to let people propose meaningful content and keeps things clean here?

Thinking about something like [this](https://github.com/ctjhoa/rust-learning)

WDYT?

**Comment by moxiegirl** [comment]
@abronan Well, I wanted to include it in the actual documentation.  I have a central private repository where we can put that.  But it won't be intended for GitHub but for docs.docker.com.  Work for you?

**Comment by abronan** [comment]
Ok, I'm working on the list at the moment (I'm half-way, trying to gather as much on every topic as I can), we can work on including this in the docs once the list is complete before the release.

**Comment by abronan** [comment]
@moxiegirl By the way I stumbled upon a currated list that we maintain for Swarm blog posts: https://github.com/docker/community/blob/master/Docker-Releases/Docker%20Swarm.m

This can be of use :ghost:

**Comment by moxiegirl** [comment]
@abronan Why don't we link to that page from the Swarm docs rather than maintaining our own? We can also work with the community to update that one.  WDYT?

**Comment by abronan** [comment]
SGTM! :)

I'm still preparing something a bit more structured though (with a table of content and sections for discovery/setup/tips/HA/TLS, etc.). I think this will help tremendously when searching for a particular topic rather than a long list of links ;)
